Dockerfile.bionic

```diff
@@ -0,0 +1,56 @@
+# Recipe of the enviornment needed to build ar.
+#
+# Author: Frankie Li, Daran He
+FROM ubuntu:bionic
+
+# Standard issued requirements for all build enviornments in Ditto. We need
+# to maintain consistent sets of numerical libraries.
+
+RUN apt-get update -qq && apt-get install -y \
```

Comment:
FYI, we removed some of these libraries from opencv build to remove the 3rd party dependencies. I think they might be existing here because we had a generic Dockerfile for all packages.
https://github.com/dittovto/ditto-opencv-builder/pull/11/files

Reply:
Ah, okay. I see that the release/v3.3 was never merged into master, grrrrr. I'm going to submit a PR to merge that into master so we can pick up those changes.

Reply:
Okay, I'll delete those packages from here, too.

Reply:
I removed those packages and dlib builds fine. I'm guessing they weren't necessary and weren't linked anyway.